lighthouse-core/gather/driver.js

```diff
@@ -464,8 +464,6 @@ class Driver {
         });
     }
 
-    log.verbose('Driver', `Installing longtask listener for CPU idle.`);
-    this.evaluateScriptOnLoad(`(${installPerformanceObserver.toString()})()`);
```

Comment:
nope, that's separate. `markAsResolvable` is for ensuring that we wait for no long tasks *after* network idle and not fire early while we're still downloading massive scripts or something